index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <title>Behnam Aghaali</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+        <link rel="stylesheet" href="index.css"/>
+    </head>
+    <body>
+        <main>
+            <nav>
+                <img src="images/logo.png" alt="logo"/>
+                <i class="fa fa-bars"></i>
```

Comment:
- [ ]  It's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of `UX` convention make the navigation in the website way more pleasant and intuitive.